deck-functional-tests/docker/build-ci.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+set -e
+set -x
+
+# Call this script from the root of the deck-functional-tests directory
+
+BUILD_TAG="deck-functional-tests:ci"
+PUBLISH_TAG="gcr.io/hebridean-sun/deck-functional-tests:ci"
```

Comment:
What is this URL? What is "hebridean-sun"?

Reply:
It's the GCP project I test work in. Once this process is solidified the CI image would ideally be available publicly for non-Google users to use.  This should be parameterized.

Reply:
I think this needs to accomodate OSS users creating their own stuff. I'm not sure what's in the image, but if I have a custom deck extension I shoudl be able to test it using this framework in which case I might have things that arent going to be in the public image.

Also there are versioning concerns here. We would want to test 1.11 and master (and eventually 1.12) independent of one another.